common/execution_context.go

```diff
@@ -319,8 +319,12 @@ func (e *ExecutionContext) EvalHandle(
 	}
 	res, err := e.eval(apiCtx, opts, js.ToString().String(), evalArgs...)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("evaluating handle: %w", err)
+	}
+	if res == nil {
```

Comment:
if we ran:

```javascript
function() {
  console.log("hello world)
}
```

This wouldn't return a result, so would `res` be `nil` in that case?

Is it possible to create a test for this scenario?

Reply:
Yes, `res` would be `nil` in that case. It's expected since `EvaluateHandle` should return a DOM handle. What would it return for that function?

Reply:
Here's a test: https://github.com/grafana/xk6-browser/pull/826/commits/c78b301967870a6033636b4bfe7d9e607ea8f78d